src/matUtils/merge.cpp

```diff
@@ -0,0 +1,131 @@
+ //Load mat files
+ //uncondense leaves
+ #include "merge.hpp"
+ po::variables_map parse_merge_command(po::parsed_options parsed) {
+    uint32_t num_cores = tbb::task_scheduler_init::default_num_threads();
+    std::string num_threads_message = "Number of threads to use when possible [DEFAULT uses all available cores, " + std::to_string(num_cores) + " detected on this machine]";
+
+    po::variables_map vm;
+    po::options_description merge_desc("merge options");
+    merge_desc.add_options()
+        ("input-mat-1,ione", po::value<std::string>()->required(),
```

Comment:
Your `ione` doesn't really work as it needed to be a single letter. You can just get rid of it.